### PR TITLE
Update BBEs under Security Category

### PR DESCRIPTION
**Pull request by ldclakmal**
## Purpose
This PR updates the following BBEs with test cases and metatags.
- crypto
- jwt-issue-validate
- url-encode-decode

Also, it adds `b7a.home` as a config in build scripts.

### CODE REVIEW

File: examples/crypto/crypto.metatags [hunk 1]
```diff
@@ -0,0 +1,2 @@
+description: BBE on how to perform different cryptographic operations such as hashing, HMAC generation, checksum generation and digitally signing data.
+keywords: ballerina, ballerina by example, bbe, crypto, hash, hmac, sign, encrypt, decrypt
```
Comment (Samallama) [inline examples/crypto/crypto.metatags:2]: Shall we reduce the keywords? The best practice is to have 5-8 keywords

File: examples/jwt-issue-validate/jwt_issue_validate.metatags [hunk 2]
```diff
@@ -0,0 +1,2 @@
+description: BBE on how to how to issue and validate a JWT.
```
Comment (Samallama) [inline examples/jwt-issue-validate/jwt_issue_validate.metatags:1]: The description has to be 50-160 characters (this is below 50)

File: examples/url-encode-decode/url_encode_decode.metatags [hunk 3]
```diff
@@ -0,0 +1,2 @@
+description: BBE on how to how to perform encoding/decoding content using different URL encoding mechanisms and algorithms.
```
Comment (Samallama) [inline examples/url-encode-decode/url_encode_decode.metatags:1]:
```suggestion
description: BBE on how to perform encoding/decoding content using different URL encoding mechanisms and algorithms in Ballerina.
```

File: examples/crypto/crypto.metatags [hunk 4]
```diff
@@ -0,0 +1,2 @@
+description: BBE on how to perform different cryptographic operations such as hashing, HMAC generation, checksum generation and digitally signing data.
```
Comment (Samallama) [inline examples/crypto/crypto.metatags:1]:
```suggestion
description: BBE on how to perform different cryptographic operations in Ballerina such as hashing, HMAC generation, checksum generation and digitally signing data.
```

File: examples/jwt-issue-validate/jwt_issue_validate.metatags (hunk 2 above)
Comment (Samallama) [inline examples/jwt-issue-validate/jwt_issue_validate.metatags:1]:
```suggestion
description: BBE on how to issue and validate a JSON Web Token (JWT) in Ballerina.
```

File: examples/crypto/crypto.metatags (hunk 1 above)
Comment (ldclakmal) [inline examples/crypto/crypto.metatags:2]: Updated with 8 keywords.
```suggestion
keywords: ballerina, ballerina by example, bbe, crypto, hash, hmac, sign, encrypt decrypt
```

Comment (ldclakmal) [reply examples/crypto/crypto.metatags]: Updated with 8 keywords. https://github.com/ballerina-platform/ballerina-distribution/pull/551/commits/f962ab67f771beb9643e82826fbf4e5282cb05ec

File: examples/jwt-issue-validate/jwt_issue_validate.metatags (hunk 2 above)
Comment (ldclakmal) [reply examples/jwt-issue-validate/jwt_issue_validate.metatags]: Updated with https://github.com/ballerina-platform/ballerina-distribution/pull/551/commits/f962ab67f771beb9643e82826fbf4e5282cb05ec

File: ballerina/build.gradle [hunk 5]
```diff
@@ -1096,12 +1095,12 @@ task testExamples() {
...
                 exec {
                     workingDir "${buildDir}/TestProject"
-                    commandLine 'cmd', '/c', "${distPath}/bin/ballerina.bat", 'build', '--experimental', "${bbe}"
+                    commandLine 'cmd', '/c', "${distPath}/bin/ballerina.bat", 'build', '--experimental', "${bbe} --b7a.home=${distPath}/jballerina-tools-${ballerinaVersion}"
                 }
             } else {
                 exec {
                     workingDir "${baseBuildPath}/TestProject"
-                    commandLine 'sh', '-c', "../jballerina-tools-${ballerinaVersion}/bin/ballerina build --experimental ${bbe}"
+                    commandLine 'sh', '-c', "${baseBuildPath}/jballerina-tools-${ballerinaVersion}/bin/ballerina build --experimental ${bbe} --b7a.home=${baseBuildPath}/jballerina-tools-${ballerinaVersion}"
```
Comment (BuddhiWathsala) [inline ballerina/build.gradle:1103]: shall we use `${distPath}/bin/ballerina` here as well?

Comment (ldclakmal) [reply ballerina/build.gradle]: Yes. Fixed all the places with https://github.com/ballerina-platform/ballerina-distribution/pull/551/commits/f51de3066bb598517eb60177e885880c11be382e

**Pull request merged**